### PR TITLE
Create Issue templates

### LINKED ISSUE #412: Issue templates
We need issue templates to collect useful information, including but not specific to:

- Asset edition (which *.exe*/*.zip*)
- Environment (OS version)
- XML configuration (in code blocks)
- Are you using Jenkins?

### PR DESCRIPTION
**Pull request by oleg-nenashev**
I took some templates from my other projects. Styling could be aligned better, but I would like to start with something.

Fixes #412

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+name: "🐛 Bug report"
+labels: bug
+about: Create a bug report to help us improve
+---
+
+**Summary**. Describe the expected and actual behavior here.
+
+## Steps to reproduce
+
+Please clarify how to reproduce the issue.
```
Comment (NextTurn) [inline .github/ISSUE_TEMPLATE/2-bug-report.md]: You may want to cover two additional points mentioned in #412:

- Mention the code block format in Markdown
A number of users don't know how to paste their XML content correctly.

- Ask whether WinSW is used with Jenkins
This helps us to triage Jenkins module related issues.

Comment (oleg-nenashev) [reply .github/ISSUE_TEMPLATE/2-bug-report.md]: Agreed

**Comment by NextTurn** [comment]
Squash?

**Comment by oleg-nenashev** [comment]
In Jenkins I commonly squash pull requests when commits in the pull request are not self-consistent. I am ready to follow this approach in WinSW

**Pull request merged**